src/e2e-test/java/io/cdap/plugin/bigquerymultitable/runners/TestRunnerRequired.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.bigquerymultitable.runners;
+
+import io.cucumber.junit.Cucumber;
+import io.cucumber.junit.CucumberOptions;
+import org.junit.runner.RunWith;
+
+/**
+ * Test Runner to execute only required BQMT action cases.
+ */
+@RunWith(Cucumber.class)
+@CucumberOptions(
+  features = {"src/e2e-test/features"},
+  glue = {"io.cdap.plugin.bigquerymultitable.stepsdesign", "io.cdap.plugin.gcs.stepsdesign", "stepsdesign",
+    "io.cdap.plugin.common.stepsdesign"},
+  tags = {"@BQMT_Required"},
+  //TODO: Enable test once issue is fixed https://cdap.atlassian.net/browse/PLUGIN-1134
```

Comment:
Can you plz help me understand how is `PLUGIN-1134` related?

I cannot see any scenario involving `GCSMove` plugin in this PR.

Reply:
Resolved